src/components/Navigation/CustomNavLink.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 
-function CustomNavLink( props ) {
+function CustomNavLink( { children, ...props } ) {
```

Comment:
I think that it's useless, use can either just take props and then deconstruct the children below:
const { children } = props;

But in this case, I would not even do that, just write props.children.

Reply:
Done